abe/fame.go

```diff
@@ -364,6 +364,10 @@ func (a *FAME) Decrypt(cipher *FAMECipher, key *FAMEAttribKeys, pk *FAMEPubKey)
 	}
 
 	// get a combination alpha of keys needed to decrypt
```

Comment:
Could you just move your check before the comment in L366, since the comment is about L367.

Reply:
fair comment (no puns intended :)) updated!